src/main/java/com/example/Library/services/AuthenticationService.java

```diff
@@ -39,6 +41,7 @@ public AuthenticationResponseDTO authenticate(@RequestBody AuthenticationRequest
             return new AuthenticationResponseDTO(request.getEmail(), token);
 
         } catch (AuthenticationException e) {
+            log.error("Invalid email/password combination : " + request.getEmail() + " " + request.getPassword());
```

Comment:
sensitive credentials logging (such as password) is a bad practice, if you still want to log it -> replace password symbols with asterisks (*)